.ci/test.sh

```diff
@@ -47,6 +52,22 @@ if [[ "${TASK}" == "r-package" ]]; then
     exit 0
 fi
 
+cd "${BUILD_DIRECTORY}"
+
+if [[ $TASK == "swig" ]]; then
```

Comment:
Moving this up avoids the need to set conda and Python-related environment variables in `swig.yml`